fe/src/main/java/org/apache/doris/load/routineload/RoutineLoadJob.java

```diff
@@ -1160,6 +1169,10 @@ public void readFields(DataInput in) throws IOException {
 
         origStmt = Text.readString(in);
 
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_59) {
+            strictMode = in.readBoolean();
```

Comment:
If you make all old routine load job to strict mode. This may cause many data to be filtered